src/main/java/org/smartregister/converters/StockConverter.java

```diff
@@ -47,7 +48,8 @@ public static Bundle convertStockToBundleResource(StockAndProductDetails stockAn
 		Reference location = Reference.builder().reference(String.of("Location/" + locationId)).build();
 
 		String serialNumber = stockAndProductDetails != null && stockAndProductDetails.getStock() != null &&
-				stockAndProductDetails.getStock().getSerialNumber() != null ?
+				stockAndProductDetails.getStock().getSerialNumber() != null &&
```

Comment:
This line is not needed now since `StringUtils.isNotBlank` or `StringUtils.isBlank` checks also for `null`